### PR TITLE
Few

### PR DESCRIPTION
**Pull request by JustEugen**
I did some changes here youpta!

### CODE REVIEW

**Review by mytriv — COMMENTED** [review]
zbs yopta!

File: index.html [hunk 1]
```diff
@@ -1 +1,10 @@
-adsf
```
Comment (mytriv) [inline index.html:1]: zbs yopta!

File: index.html [hunk 2]
```diff
@@ -1 +1,10 @@
...
+A
```
Comment (mytriv) [inline index.html:1]: im gay

Comment (JustEugen) [reply index.html:1]: > im gay

yes me too

**Review by mytriv — APPROVED** [review]

**Pull request merged**